sdk/openai/openai/sources/customizations/models/audio.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { OperationOptions } from "@azure-rest/core-client";
+
+export type AudioResultFormat =
+  /** This format will return an JSON structure containing a single \"text\" with the transcription. */
+  | "json"
+  /** This format will return an JSON structure containing an enriched structure with the transcription. */
+  | "verbose_json"
+  /** This will make the response return the transcription as plain/text. */
+  | "text"
+  /** The transcription will be provided in SRT format (SubRip Text) in the form of plain/text. */
+  | "srt"
+  /** The transcription will be provided in VTT format (Web Video Text Tracks) in the form of plain/text. */
+  | "vtt";
+
+/** Lorem ipsum */
```

Comment:
Updated comment for the interface
```suggestion
/** Simple transcription response */
```